### ISSUE TITLE
Crash Twinsanity - Graphical Issues (Steam Deck)

### ISSUE THREAD

**Issue by TTRaven** [xemu-project/xemu#1371]
### Title

https://xemu.app/titles/56550036/#Crash-Twinsanity

### Bug Description

HUD elements flash on screen, and flash the animation over and over again but there is no game play shown, just a black screen.

No steps, just loading up the game as normal. (I have verified this iso works on my windows machine)

https://user-images.githubusercontent.com/54518605/214958846-ce3e2a6d-880a-4731-9c26-1c1d99e0acc7.mp4

### Expected Behavior

Should expect game play to be shown, and the UI to not flicker as it does.

### xemu Version

Version:      0.7.83
Branch:       ca6e712ecd797c77ba501402cfcd64dde55d2c17
Commit:       ca6e712ecd797c77ba501402cfcd64dde55d2c17
Date:         Wed Jan 18 11:24:04 PM UTC 2023

### System Information

CPU:          AMD Custom APU 0405
OS Platform:  Linux
OS Version:   Freedesktop.org SDK 22.08 (Flatpak runtime)
Manufacturer: AMD
GPU Model:    AMD Custom GPU 0405 (vangogh, LLVM 14.0.6, DRM 3.45, 5.13.0-valve36-1-neptune)
Driver:       4.6 (Core Profile) Mesa 22.3.2 (git-a09d5e2747)
Shader:       4.60

### Additional Context

I have verified that it is not my rip of the iso, game does not have this issue on my windows machine.

**Comment by TTRaven** [comment]
oops guess i forgot to change the title... my bad. Can repost if needed.

**Comment by mborgerson** [comment]
> oops guess i forgot to change the title... my bad. Can repost if needed.

@TTRaven you can edit the title. Click the edit button to the right of the title.

**Comment by TTRaven** [comment]
I did end up installing windows onto my steam deck to test this, and it did resolve the issue, so it is only a thing on linux.
But it does bring up some new issues, I keep on getting black screen lock ups, may just be a setting on my end, will update if I find a fix

**Comment by capezotte** [comment]
Can reproduce this issue on an Intel GPU.

https://github.com/xemu-project/xemu/assets/7990860/e8181d55-e93c-4033-88c8-66fde1a5733f

I've also noticed the fog effect during the official intro animation is missing, maybe that's related? 

However, it seems to be fixed (both fog in the intro and gameplay graphics) when running the dedicated AMD GPU on my laptop.

### Xemu version
xemu_version: 0.7.117
xemu_branch: b605381adb3d13ad5b1be94d06bcfb896590d90a
xemu_commit: b605381adb3d13ad5b1be94d06bcfb896590d90a
xemu_date: Sun Nov 19 09:23:11 PM UTC 2023

### System Info
CPU: Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
OS_Version: Freedesktop.org SDK 23.08 (Flatpak runtime)

### Intel GPU (has graphical issues)
GL_VENDOR: Intel
GL_RENDERER: Mesa Intel(R) UHD Graphics 620 (WHL GT2)
GL_VERSION: 4.6 (Core Profile) Mesa 23.2.1 (git-49a47f187e)
GL_SHADING_LANGUAGE_VERSION: 4.60

### AMD GPU (no graphical issues)
GL_VENDOR: AMD
GL_RENDERER: AMD Radeon R5 M435 (hainan, LLVM 16.0.6, DRM 3.54, 6.5.7-xanmod1)
GL_VERSION: 4.6 (Core Profile) Mesa 23.2.1 (git-49a47f187e)
GL_SHADING_LANGUAGE_VERSION: 4.60

**Comment by Ry0n4Irv** [comment]
yeah im having an identical issue, still classed as playable on xemu site, was there a fix for this?

**Comment by Triticum0** [comment]
dupe of #658

**Comment by Triticum0** [comment]
@TTRaven @capezotte Retest it should be fixed. test on my intel 5000 series gpu and seems fixed.
